.github/workflows/devel.yaml

```diff
@@ -0,0 +1,39 @@
+name: Devel
+on:
+  schedule:
+    - cron: '0 4 * * *'
```

Comment:
we have used e2e "nightly" trigger to follow e2e is "alive" but AFAICS with this modification we end up doing a nightly push too?

Reply:
That's correct. I can re-create the e2e workflow and set it to run periodically, and remove the schedule from devel.